tests/app/views/test_suppliers.py

```diff
@@ -1320,6 +1320,136 @@ def test_get_supplier_framework_info(self, open_g8_framework):
             'uploaderUserId': 20
         }
 
+    def test_get_supplier_framework_returns_framework_agreement(self, live_example_framework):
```

Comment:
It feels like there should be some testing around always getting the most recent signed agreement details when there are multiple signed agreements saved.